src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
+import static java.lang.Math.abs;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 32;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
 
+        int number = getNubmer(key);
+        Node<K,V> newNode = new Node<>(key,value);
+        Node<K,V> currentNode = table[number];
+        if (table[number] == null) {
+            table[number] = newNode;
+            size++;
```

Comment:
extract  common part from if

Reply:
not fixed. 
`size++` is the common part. Put it after if-else